strategy/strategy04_points.php

```diff
@@ -156,7 +168,7 @@ public function validation($data, $files) {
         $ratings = $data ['data'];
         $currentpoints = 0;
         foreach ($ratings as $cid => $rating) {
-            if ($rating['rating'] < 0 || $rating['rating'] > $totalpoints) {
+            if ($rating['rating'] < 0 || $rating['rating'] > $totalpoints || $rating['rating'] > $maxperchoice) {
                 $errors['data[' . $cid . '][rating]'] = get_string(strategy::STRATEGYID . '_illegal_entry', ratingallocate_MOD_NAME, $totalpoints);
```

Comment:
This error message should be changed to maxperchoice if this is smaller than totalpoints (what should be the usecase).